src/util/simplify_expr.cpp

```diff
@@ -1365,7 +1365,8 @@ bool simplify_exprt::simplify_update(exprt &expr)
     {
       const irep_idt &component_name=
         e.get(ID_component_name);
-      const struct_typet &value_ptr_struct_type=to_struct_type(value_ptr_type);
+      const struct_typet &value_ptr_struct_type =
+        to_struct_type(value_ptr_type);
```

Comment:
Please squash with the previous commit, the clang-format changes are restricted to code that you have introduced in the preceding commit.